### DISCUSSION TITLE
lolMiner 1.65

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1820] [General]
### Changes
- Improved Kaspa performance in ETC and ETHW dual mining by about 6-7% on AMD (Big) Navi GPUs and 10-12% on Nvidia Turing and Ampere, measured at the same Ethash speed. The actual improvement depends on the concretely selected dual factor. Re-tuning your setup is highly recommended.
- Improved Kaspa performance in Kaspa only mining mode by 0.6 to 1.2% on Nvidia GPUs and 0.4-0.5% on AMD (Big) Navi GPUs at approximately same power draw.
-  Beta feature: Added experimental support for AMD RX7900 series. Supported algorithms: Et(c)hash, kHeavyHash and the corresponding dual mining.  

### Fixes
- Fixed some minor display bugs 

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.65'>lolMiner 1.65</a>.</em>

**Comment by Cybertiron** [comment]
Lol 1st to see this!

**Comment by nirotub** [comment]
На Kaspa стейлов стало много. При тех же настройках разгона

**Reply by Lolliedieb** [reply]
There should not be more then for any previous version.... can you  share me more details about your system configuration?

**Reply by nirotub** [reply]
HiveOS
1х RTX 2060 Super (Core 1560, Mem 2000, PL 170)
2x RTX 3060 Ti (Core 1680, Mem 2900, PL 180)

**Reply by VDV777** [reply]
Это скорее всего вина пула, а не майнера или вашего провайдера, майнер - это всего лишь мостик между пулом и клиентом. Хотя 35мс это вполне хороший пинг (из вашего скрина). Попробуйте поменять адрес пула, на wooly pooly наверняка он не один и даже не два)

**Comment by nirotub** [comment]
![Безымянный](https://user-images.githubusercontent.com/47082199/209080580-d444a9b5-8723-4e02-bc4d-8c2f7339e1c9.png)

**Reply by Lolliedieb** [reply]
Thanks for reporting - I will investigate. That is indeed more then it should be at that ping to pool.

**Comment by nirotub** [comment]
На втором риге вот так
![Безымянный](https://user-images.githubusercontent.com/47082199/209143800-2b5ca41a-63ce-4f38-8ebd-686e868fe662.png)

**Comment by Katriox** [comment]
Dual mining not working with nicehash, mining freezes after some minutes and stop mining with nothing on the log, i tried without overclocks and it's the same 

![image](https://user-images.githubusercontent.com/121240573/209150279-47845363-d452-4632-8585-8f1ecb778d83.png)

**Comment by VDV777** [comment]
pls fix it.
![1](https://user-images.githubusercontent.com/60316206/209304967-70f14d13-63d4-401f-a610-158b51536507.png)

**Comment by VDV777** [comment]
And bug number 2:
If we have written in bat file command: `set "EXTRAPARAMETERS=--apiport 8020 --cclk 1400,1450,1450, *, 1450 --mclk 810,810,810,*, 810, --pl 135, 135, 135, *, 140"`. Command to apply for the first, second and third video adapter, but not for fifth video adapter. Program has been try accept for the fourth adapter, but fourth adaper is amd ext graphics and nothing is happening... Only three adapters have been changed.
Bug number 3:
Command -pl hasn't been accept evrything, as admin privelleges or not. p.s. windows 10/11 64 last upd, nvidia drivers ver. on screenshot

**Comment by Lolliedieb** [comment]
Might be cause --devices did not include the amd... then please also keep
oc options to selected cards

VDV777 ***@***.***> schrieb am Fr., 23. Dez. 2022, 10:18:

**Reply by VDV777** [reply]
I mean, if you have amd ext graphic or intel, and marked it `*`, a programm will stop, and not to be apply settings for video adapters before `*`. If you have five videoadapters: 
`videocards, videocards, videocards, ext graphic, videocards`
and use `--cclk 1400,1450,1450, *, 1450 --mclk 810,810,810,*, 810`
the fourth videocard will not get a settings, but if we will use `--cclk 1400,1450,1450, 1450 --mclk 810,810,810,810` - it's fine!

**Comment by XdR2323** [comment]
hello, my mining equipment, which works fine in 1.64 version, is problematic for problematic amd cards in 1.65 and 1.65a, for some reason it does not mine caspas
In the 1.65 version, it does it for 3 minutes at low speed, then goes to n/a error and my amd cards suck.
I said it would be fixed in 1.65a, but it didn't, the only thing that fixed it was to continue digging etc.
  How can we solve this problem? can you help me?
There is no OC setting that I haven't tried, and I even ran it at stock values, I didn't enter any OC, the problem still persisted. Nvidia's are fine, but I don't understand why amds are like that.
![fgt](https://user-images.githubusercontent.com/121347833/209439668-372af5f1-af1b-4735-8b0e-368003272238.JPG)

**Comment by Regiven123** [comment]
**Microsoft Defender find a Virus "CoinMiner" WARNING!!!**

**Comment by Lolliedieb** [comment]
Think for a moment... this is supposed to be a damn coin miner! Its not
really telling us anything new...

Regiven123 ***@***.***> schrieb am Mi., 4. Jan. 2023, 17:20:

**Comment by Regiven123** [comment]
After starting the miner under Windows, and after closing the miner, the settings in the video card are fixed, you can’t use the video card as usual, it slows down terribly, in order to fix this, every after the miner you need to restart the PC, you can fix it somehow, do something after closing the miner, Did the video card work as usual?

**Comment by Lolliedieb** [comment]
Make sure you close the miner via ctrl+c and not by closing the Bash window
by hitting the x. On ctlr+c the overclock settings will be fully reset.

Am Mi., 8. Feb. 2023 um 21:38 Uhr schrieb Regiven123 <
***@***.***>:
